{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"apiError"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"?"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":2,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/dfc\/dfcawserviceerror\/apierror"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.deserve.dfc\/documentation\/DFC\/DFCAWServiceError\/apiError","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"This is an actual error that has occured from an inner API."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"apiError"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"?"}],"title":"apiError","roleHeading":"Instance Property","role":"symbol","symbolKind":"property","externalID":"s:3DFC17DFCAWServiceErrorV03apiC0s0C0_pSgvp","modules":[{"name":"DFC"}]},"hierarchy":{"paths":[["doc:\/\/com.deserve.dfc\/documentation\/DFC","doc:\/\/com.deserve.dfc\/documentation\/DFC\/DFCAWServiceError"]]},"references":{"doc://com.deserve.dfc/documentation/DFC":{"role":"collection","title":"DFC","abstract":[],"identifier":"doc:\/\/com.deserve.dfc\/documentation\/DFC","kind":"symbol","type":"topic","url":"\/documentation\/dfc"},"doc://com.deserve.dfc/documentation/DFC/DFCAWServiceError/apiError":{"role":"symbol","title":"apiError","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"apiError"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"?"}],"abstract":[{"type":"text","text":"This is an actual error that has occured from an inner API."}],"identifier":"doc:\/\/com.deserve.dfc\/documentation\/DFC\/DFCAWServiceError\/apiError","kind":"symbol","type":"topic","url":"\/documentation\/dfc\/dfcawserviceerror\/apierror"},"doc://com.deserve.dfc/documentation/DFC/DFCAWServiceError":{"role":"symbol","title":"DFCAWServiceError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"DFCAWServiceError"}],"abstract":[{"type":"text","text":"Class which holds information about error that may occur during push provisioning."}],"identifier":"doc:\/\/com.deserve.dfc\/documentation\/DFC\/DFCAWServiceError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"DFCAWServiceError"}],"url":"\/documentation\/dfc\/dfcawserviceerror"}}}